client/scala/src/main/scala/ai/verta/blobs/dataset/PathBlob.scala

```diff
@@ -27,28 +31,42 @@ case class PathBlob(protected val contents: HashMap[String, FileMetadata]) exten
 
 /** Companion object to initialize instances and handle interaction with versioning blob */
 object PathBlob {
-  private val BufferSize = 8192
-
   /** Constructor taking only one path
+   *  @param path a single path
+   *  @param enableMDBVersioning whether to version the data in the blob
+   *  @return if the path is invalid, a failure along with exception message. Otherwise, the pathblob (wrapped in success)
+   */
+  def apply(path: String, enableMDBVersioning: Boolean): Try[PathBlob] = apply(List(path), enableMDBVersioning)
+
+  /** Constructor taking only one path. Does not version the data in the blob with ModelDB
    *  @param path a single path
    *  @return if the path is invalid, a failure along with exception message. Otherwise, the pathblob (wrapped in success)
    */
-  def apply(path: String): Try[PathBlob] = apply(List(path))
+  def apply(path: String): Try[PathBlob] = apply(List(path), false)
```

Comment:
```suggestion
  def apply(path: String): Try[PathBlob] = apply(List(path))
```
I recommend doing one step at a time

Reply:
Fixed.